go.mod

```diff
@@ -160,5 +160,5 @@ require (
 replace (
 	github.com/cosmos/cosmos-sdk => github.com/celestiaorg/cosmos-sdk v1.2.0-sdk-v0.46.0
 	github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
-	github.com/tendermint/tendermint => github.com/celestiaorg/celestia-core v1.5.0-tm-v0.34.20
+	github.com/tendermint/tendermint v0.34.20 => github.com/celestiaorg/celestia-core v1.5.0-tm-v0.34.20
```

Comment:
[question][non-blocking] out of curiosity, why is the exact tendermint version specified?

Reply:
whoops, good catch, this actually wasn't on purpose. While it technically shouldn't matter we might want to switch it back. I think the only time it did matter was when used to change the imports from `tendermint/tendermint` to `celestiaorg/celestia-core`

Reply:
merging w/o as this is optional, and I don't want to look at this PR anymore lol

Reply:
Agreed. :shipit: and we can always remove later